src/libraries/System.Private.CoreLib/src/System/SearchValues/SearchValues.T.cs

```diff
@@ -80,5 +81,24 @@ internal static int LastIndexOfAnyExcept(ReadOnlySpan<T> span, SearchValues<T> v
 
             return values.LastIndexOfAnyExcept(span);
         }
+
+        private string DebuggerDisplay
+        {
+            get
+            {
+                T[] values = GetValues();
+
+                string display = $"{GetType().Name}, Count={values.Length}";
```

Comment:
I'm using this in my testing, helps a ton when there are multiple generic variants.

```c#
Type implType = searchValuesInstance.GetType();
string impl = $"{implType.Name} [{string.Join(", ", implType.GenericTypeArguments.Select(t => t.Name))}]";
```

E.g.
``SingleStringSearchValuesThreeChars`2 [ValueLength8OrLonger, CaseSensitive]``